amnesia/src/CMakeLists.txt

```diff
@@ -8,71 +8,71 @@ option(USERDIR_RESOURCES "Allow using local userdirectory resources" ON)
 option(FBX_SUPPORT "Add FBX support during compilation" OFF)
 
 set(CMAKE_BUILD_TYPE "Release")
-# Release: high optimization level, no debug info, code or asserts.
-# Debug: No optimization, asserts enabled, debug info included in executable (so you can step through the code with a debugger and have address to source-file:line-number translation).
-# RelWithDebInfo: optimized, *with* debug info, but no debug (output) code or asserts.
-# MinSizeRel: same as Release but optimizing for size rather than speed.
+# Release: High optimization level, no debug info, no debug code, no asserts.
+# Debug: No optimization, asserts enabled, debug info included (enables stepping through code with a debugger and source-file:line-number translation).
+# RelWithDebInfo: Optimized with debug info, but no debug output code or asserts.
```

Comment:
This makes it sound like debug info is something that optimizes programs

Reply:
i.e. "optimized with debug info"

Reply:
Done.